src/features/validate/handler.ts

```diff
@@ -33,13 +34,23 @@ export async function validateRegistrationHandler(
       headers: request.headers
     })
   } catch (err) {
+    /*
+
+    I think that there might be a reason why we still call this endpoint even on an error
+    But I think it is strange so I commented this out.
+
+    I dont know what effect it is having yet
+
     fetch(CONFIRM_REGISTRATION_URL, {
       method: 'POST',
       body: JSON.stringify({ error: err.message }),
       headers: request.headers
-    })
+    })*/
```

Comment:
@rabiulislamanik we need to investigate why we call this endpoint even on an error.  Makes no sense to me